### PR TITLE
DONT MERGE - Stage 3

### PR DESCRIPTION
**Pull request by mfix22**

### CODE REVIEW

File: app/components/Settings/cron-picker/cron-picker.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.mode-picker {
```
Comment (briandennis) [inline app/components/Settings/cron-picker/cron-picker.css]: we can remove rules we're not using

File: app/components/Settings/cron-picker/cron-picker.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import {Row} from '../../layout.jsx';
+
+// react-select requires options to have `label` and `value` keys
+const MODE_OPTIONS = MODES.map(currMode => ({
+    ...currMode,
+    label: currMode.name,
+    value: currMode.id
+}));
+
+const rowStyle = {
```
Comment (briandennis) [inline app/components/Settings/cron-picker/cron-picker.jsx]: can we move this to the css file?

File: app/components/Settings/cron-picker/details-row.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Row} from '../../layout.jsx';
+
+const rowStyle = {justifyContent: 'flex-start'};
```
Comment (briandennis) [inline app/components/Settings/cron-picker/details-row.jsx]: pull into css file

File: app/components/Settings/cron-picker/modes/daily.jsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        };
+
+        props.onChange(this.toCronExpression(props.initialTime));
+
+        this.onChange = this.onChange.bind(this);
+    }
+
+    toCronExpression({hour, minute, amPm}) {
+        // adjust hour to fit cron format if needed
+        let normalizedHour = hour;
```
Comment (briandennis) [inline app/components/Settings/cron-picker/modes/daily.jsx]: isn't there a util method for this? (silly past Brian)

File: app/components/Settings/scheduler/preview-modal.jsx [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
+import {Controlled as CodeMirror} from 'react-codemirror2';
 import ms from 'ms';
 
 import Modal from '../../modal.jsx';
 import Error from '../../error.jsx';
-import { Link } from '../../Link.react.js';
-import { FrequencySelector } from './create-modal.jsx';
-import { Row, Column } from '../../layout.jsx';
+import {Link} from '../../Link.react.js';
+import {DEFAULT_REFRESH_INTERVAL} from './create-modal.jsx';
```
Comment (briandennis) [inline app/components/Settings/scheduler/preview-modal.jsx]: is there another constants location we could be storing this?

File: app/components/Settings/cron-picker/details-row.jsx (hunk 3 above)
Comment (mfix22) [reply app/components/Settings/cron-picker/details-row.jsx]: No.

**Pull request closed without merging**